packages/core/nuxt-theme-module/index.js

```diff
@@ -19,58 +19,78 @@ const log = {
 module.exports = async function DefaultThemeModule(moduleOptions) {
   log.info(chalk.green('Starting Theme Module'));
 
-  const agnosticThemeDir = path.join(__dirname, 'theme');
-  const projectLocalThemeDir = this.options.buildDir.replace('.nuxt', '.theme');
+  if (typeof moduleOptions.routes === 'undefined') {
```

Comment:
**code organization** I think it would be better to just have a default config and merge it with provided one (via `lodash/merge`). Otherwise we will have to make this check for each "default" property value

Reply:
I agree

Reply:
Done